src/components/swap/ConfirmSwapModal.tsx

```diff
@@ -43,7 +43,7 @@ import SwapModalHeader from './SwapModalHeader'
 export enum ConfirmModalState {
   REVIEWING,
   WRAPPING,
-  RESETTING_USDT,
+  RESETTING_TOKEN,
```

Comment:
nit: would resetting_token_allowance make more sense?

Reply:
it woudl